src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/TableSchemaObject.java

```diff
@@ -115,4 +128,121 @@ public static TableSchemaObject from(TableMetadata tableMetadata, ObjectMapper o
     }
     return new TableSchemaObject(tableMetadata, vectorConfig);
   }
+
```

Comment:
there are a number of things to imprve in the from() function https://github.com/stargate/data-api/issues/1521